source/topics/troubleshooting/what-is-a-csr.md

```diff
@@ -0,0 +1,19 @@
+A CSR, or [Certificate Signing Request](https://en.wikipedia.org/wiki/Certificate_signing_request), is a message you generate with information about the SSL/TLS certificate you want your SSL/TLS certificate provider to issue. You encrypt the CSR with your private key.
```

Comment:
There's a lot of misinformation about this on the internet, but the CSR isn't encrypted. The Wikipedia article you're linking actually includes an ASN 1 dump (i.e. a readout of the data in a CSR) of a typical CSR, which as you can see isn't encrypted (if it were, you'd need to send a CSR and your public key separately).

Instead, you include your public key into the CSR (I'm not 100% sure whether you include the actual key or a digest TBH).

Maybe @fancyremarker has a few more details I'm missing.